config/initializers/redis_config.rb

```diff
@@ -1,12 +1,9 @@
-rails_root = ENV['RAILS_ROOT'] || File.dirname(__FILE__) + '/../..'
-rails_env = ENV['RAILS_ENV'] || 'development'
-
-# use config/redis.yml to load settings
-redis_config_path = "#{rails_root}/config/redis.yml"
-redis_config = YAML.load(ERB.new(IO.read(redis_config_path)).result)
+require 'redis'
 
 # initialize redis connection
-Resque.redis = redis_config[rails_env]
-
-# pull out the parsed redis driver from resque
-$redis = Resque.redis.instance_eval{@redis}
+if ENV['REDIS_URL']
+  $redis = Redis.new
+else
+  # use config/secrets.yml to load settings
+  $redis = Redis.new(:url=> Rails.application.secrets.redis_url)
```

Comment:
Thanks. And learnt something new re: [ruby hash style](https://github.com/github/rubocop-github/blob/master/STYLEGUIDE.md#hashes). Also, I'm curious, what is the loading order of the initializers .rb files when the app is initialized? alphebatical order? i.e. Redis.current (or $redis) is initialized before loading resque_config.